docs/advisories/cve_2017_14491.md

```diff
@@ -0,0 +1,129 @@
+# CVE-2017-14491
+
+A vulnerability in dnsmasq, used by kube-dns, requires an upgrade to the kube-dns component. This component is the default DNS component installed in Kubernetes.  The vulnerability may be externally exploitable. Links below exist with the full detail
+of the CVE. This is not a Kubernetes specific vunerability, but exists in dnsmasq.
+
```

Comment:
How about  a link here to the "hotfix instructions" which says "if you just want to fix a running cluster, or if you are not using kops, there is a simple procedure to manually update kube-dns"